### PR TITLE
xz FeatureModel Update

### PR DESCRIPTION
**Pull request by Keller-1998**
Updates Locations of xz FeatureModel to xz commit 4773608.

### CODE REVIEW

File: xz/FeatureModel.xml [hunk 1]
```diff
@@ -1721,11 +1721,11 @@
         <sourceRange>
           <path>src/xz/hardware.c</path>
           <start>
-            <line>21</line>
+            <line>25</line>
             <column>17</column>
           </start>
           <end>
-            <line>21</line>
+            <line>25</line>
```
Comment (vulder) [inline xz/FeatureModel.xml:1728]: Why did these numbers change? Did you pick a new/other version?

**Review by vulder — CHANGES_REQUESTED** [review]

File: xz/FeatureModel.xml (hunk 1 above)
Comment (Keller-1998) [reply xz/FeatureModel.xml:1728]: Yes, I used the newest commit of xz.

Comment (vulder) [reply xz/FeatureModel.xml:1728]: Which one did you use?

We are currently defaulting to `4773608554`, in theory we enable the multi version FeatureModel feature. I'm just not sure if that's currently necessary.

Comment (Keller-1998) [reply xz/FeatureModel.xml:1728]: I also used commit `4773608554`. The locations of this commit don't fit with the locations that are currently used in the FeatureModel, for these few features.

Comment (vulder) [reply xz/FeatureModel.xml:1728]: Ok, then we should update the lines :+1:

**Review by vulder — APPROVED** [review]

**Pull request merged**